lib/cache-utils.js

```diff
@@ -9,20 +9,16 @@ export const State = {
   CacheMatchedKey: "matched_key",
 };
 
-export function getCacheDirectory() {
-  const platform = os.platform();
-  const home = process.env.HOME;
-
-  switch (platform) {
-    case "linux":
-      return path.join(home, ".cache/scarb");
-    case "darwin":
-      return path.join(home, `Library/Caches/com.swmansion.scarb`);
-    case "win32":
-      return path.join(process.env.APPDATA, "swmansion/scarb/config");
-    default:
-      throw new Error(`caching is not available on this platform: ${platform}`);
+export async function getCacheDirectory() {
```

Comment:
Oh just caught this. Can you please add `await` to all invocations of this function?